### PR TITLE
Fix some commands

### PR DESCRIPTION
**Pull request by Bryan-0**
- Fix /hourmute command, because when someone use the command without a target, mute help comes and not hourmute help. 
- Add (!this.canBroadcast()) in /roomalias command.

### CODE REVIEW

File: commands.js [hunk 1]
```diff
@@ -440,6 +440,7 @@ var commands = exports.commands = {
 	roomalias: function (target, room, user) {
 		if (!room.chatRoomData) return this.sendReply("This room isn't designed for aliases.");
 		if (!target) {
+			if (!this.canBroadcast()) return;
```
Comment (Rhythms88) [inline commands.js]: Why would anyone need to broadcast a room alias? This command is for making them.

Comment (Bryan-0) [inline commands.js]: Yes, and for showing them too.

**Comment by AustinXII** [comment]
I dont think showing the room alias is needed, I have to agree with Rhythms on this one.

**Comment by xfix** [comment]
Those changes should have probably been done separately. That said, I'm curious if you have tested this change, considering Travis complains about unknown `target` variable, as `hourmute` doesn't define usual function arguments (this shouldn't have worked at all).

Also, I don't see a problem with being able to broadcast room aliases (you can broadcast more useless things like version, but why stop that when it's consistent?).

Not merging, because this change will cause exceptions on a server when `/hourmute` is used, but feel free to improve your pull request.

File: commands.js [hunk 2]
```diff
@@ -835,6 +836,7 @@ var commands = exports.commands = {
 
 	hm: 'hourmute',
 	hourmute: function () {
```
Comment (Zarel) [inline commands.js]: You're missing `target` here.

**Comment by Bryan-0** [comment]
Updated and Fixed.

File: commands.js [hunk 3]
```diff
@@ -834,7 +835,8 @@ var commands = exports.commands = {
 	mutehelp: ["/mute OR /m [username], [reason] - Mutes a user with reason for 7 minutes. Requires: % @ # & ~"],
 
 	hm: 'hourmute',
-	hourmute: function () {
+	hourmute: function (target, room, user) {
```
Comment (kotarou3) [inline commands.js]: Why did you need to introduce `room` and `user`?

**Comment by Bryan-0** [comment]
Done.

**Comment by Zarel** [comment]
Very vague commit summary. In the future, you might want to add lines about exactly what you've changed.

**Comment by Bryan-0** [comment]
Alright.
